charts/policy-service/values.yaml

```diff
@@ -25,10 +26,10 @@ image:
 resources:
   limits:
     cpu: 750m
-    memory: 2.5Gi
+    memory: 1Gi
```

Comment:
Not sure of the reason for the change here - I presume just that we expect the policy service to be able to run in 1GB rather than 2.5GB.

Reply:
yes